static/app/views/performance/traceDetails/content.tsx

```diff
@@ -357,13 +360,15 @@ class TraceDetailsContent extends React.Component<Props, State> {
       index,
       numberOfHiddenTransactionsAbove,
       traceInfo,
+      hasGuideAnchor = true,
```

Comment:
Minor nit.. With the default I think you are better either removing the undefined type on the `hasGuideAnchor` prop or switching the prop to be `isGuideAnchorDisabled` so it doesn't need a default and you can leave it undefined minimizing the function param surface area. Just an aside though, this _might_ simplify the logic in a few places (like with disabled props to the guideanchor below, etc.)

Reply:
i removed the `undefined` type on this and made it required.